lib/decision_review_v1/service.rb

```diff
@@ -226,11 +226,11 @@ def get_notice_of_disagreement_upload(guid:)
       end
     end
 
-    def self.file_upload_metadata(user)
+    def self.file_upload_metadata(user, backup_zip = nil)
       {
```

Comment:
Only change I might suggest is to make `backup_zip` a named parameter, since on the surface it doesn't seem to have a lot of context for a method named `file_upload_metadata` but this is by no means a blocker.

Reply:
I have considered a named parameter here instead. I often use named parameters, just did not feel right here for some reason, but it can always be changed for sure.